CHANGELOG.md

```diff
@@ -12,51 +12,92 @@ You can read the full documentation for this release on [Read the Docs](http://m
 
 Because the version of typeshed we use in mypy 1.9 doesn't support 3.7, neither does mypy 1.9. (Jared Hance, PR [16883](https://github.com/python/mypy/pull/16883))
 
-#### Basic PEP 696 Support
+We are planning to enable *local partial types* (enabled via the
```

Comment:
Can we add an explanation (or link to one) of what "local partial types" are? I imagine to most people these words don't mean much.

Reply:
Maybe link to https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-local-partial-types which has a good example.